backend/domain/userDomain.ts

```diff
@@ -0,0 +1,22 @@
+import { RowDataPacket } from "mysql2";
+
+export interface IUser extends RowDataPacket {
+  id: number;
+  name: string;
+  locations: string;
+}
+
+class User {
+  id: number;
+  name: string;
+  locations: string[];
+
+  constructor(user: IUser) {
+    const { id, name, locations } = user;
+    this.id = id;
+    this.name = name;
+    this.locations = locations.split(",");
```

Comment:
```suggestion
    this.locations = locations.split(",").map((location) => location.trim());
```

혹시 모르니 앞 뒤 공백을 제거하면 어떨까도 싶습니다 ㅎㅎ